code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -16,6 +16,27 @@
 				/obj/item/seeds/sunflower/moonflower = 8
 				)
 
+//Vault strange seed disk
+
+/obj/item/weapon/disk/design_disk/strange_seed
+	name = "Lifegiver's Blessing"
+	desc = "There is some sort of strange emblem of a tree bearing growths on the disk"
+	icon_state = "datadisk1"
+	max_blueprints = 1
+
+/obj/item/weapon/disk/design_disk/strange_seed/Initialize()
+	. = ..()
+	var/datum/design/strange_seed/G = new
+	blueprints[1] = G
+
+/datum/design/strange_seed
+	name = "Strange Seed"
+	id = "seeds_random"
+	build_type = BIOGENERATOR
+	materials = list(MAT_BIOMASS = 5000)
+	build_path = /obj/item/seeds/random
+	category = list("initial", "Imported")
```

Comment:
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/research/research.dm#L170
If a design has "initial" in the category var and the BIOGENERATOR flag in build_type, it will appear at roundstart in all biogenerators. Notice how the golem shell design, right below this, does not have "initial" in its category var.